src/inmanta/deploy/scheduler.py

```diff
@@ -311,16 +311,11 @@ async def _initialize(self) -> None:
             increment: set[ResourceIdStr]
             increment, _ = await ConfigurationModel.get_increment(self.environment, version, connection=con)
 
-        resources_to_deploy: Mapping[ResourceIdStr, ResourceDetails] = {rid: resources[rid] for rid in increment}
-        up_to_date_resources: Mapping[ResourceIdStr, ResourceDetails] = {
-            rid: resources[rid] for rid in resources.keys() if rid not in increment
-        }
-
         await self._new_version(
             version,
-            resources=resources_to_deploy,
+            resources=resources,
             requires=requires,
-            up_to_date_resources=up_to_date_resources,
+            up_to_date_resources=resources.keys() - increment,
```

Comment:
this also happens to be the second return value of `get_increment`

Reply:
Right, will fix.